src/App/Controls/LabeledValueCell.cs

```diff
@@ -114,6 +114,23 @@ public class LabeledValueCell : ExtendedViewCell
 
                 containerStackLayout.AdjustPaddingForDevice();
             }
+            else if (Device.RuntimePlatform == Device.UWP)
+            {
+                buttonStackLayout.Spacing = 0;
+
+                if (Button1 != null)
+                {
+                    Button1.BackgroundColor = Color.Transparent;
+                    Button1.BorderWidth = 0;
+                    Button1.Scale = 0.5;
```

Comment:
I don't think we want to scale the button. we just need to use the proper sized images as discussed earlier

Reply:
I was a little worried how that would look on high res screens, as UWP doesn't have dynamic image selection like Android and iOS (at least not with Xamarin), but I'm not sure how well scale will work either on high res screens.